src/main/java/sysc4806/graduateAdmissions/model/User.java

```diff
@@ -0,0 +1,53 @@
+package sysc4806.graduateAdmissions.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.ArrayList;
+/**
+ * Represents one of three possible users in the system (Student, Professor, or Admin)
+ * Used to user details, credentials, and information.
+ *
+ * @author Kevin Sun
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password; 
+    private Role role;
+    private ArrayList<Interest> interests;
+    private ArrayList<String> applications; 

```

Comment:
Same problem Madelyn had. This is the fix:
![image](https://user-images.githubusercontent.com/32455969/76276797-3a6e2900-625d-11ea-96db-9ee27ccd208f.png)
Be sure to update the applications list to have the right annotation and be a Set instead of a List if her branch is merged first.